.swiftlint.yml

```diff
@@ -1,27 +1,48 @@
 disabled_rules:
-  - force_cast
-  - function_body_length
-  - function_parameter_count
+  - identifier_name
   - opening_brace
-  - type_name
-  - variable_name
 opt_in_rules:
   - conditional_returns_on_newline
-  - redundant_nil_coalesing
+  - private_outlet
+  - sorted_imports
+trailing_comma:
+  mandatory_comma: true
 line_length:
   - 110
 type_body_length:
   - 350
+private_over_fileprivate:
+  validate_extensions: true
+deployment_target:
+  iOS_deployment_target: 8.0
```

Comment:
this calls for a deployment target of 8.0 but the podfile in a later part of change indicates 9.0?  is this intentional?

Reply:
Yes. SDK has deployment target 8.0 (The example project has 9.0). So, I chose 8.0 for the swiftlint checks, because they apply to SDK code, too.